middleware/2fa.js

```diff
@@ -234,7 +250,7 @@ const initCode = async (ctx) => {
     }
 
     const { kind, detail } = method;
-    if (kind !== TWO_FACTOR_AUTH_KINDS.EMAIL) {
+    if (process.env.NODE_ENV !== 'test' && kind !== TWO_FACTOR_AUTH_KINDS.EMAIL) {
```

Comment:
We need to be able to create SMS 2FA methods in `test` so we can test verification

Reply:
Re-enabled this and instead created/using a backdoor for creating 2fa phone methods. That way we can keep this disabled and have a test backing it which ensures it's disabled across all envs.